src/main/java/org/jabref/gui/cleanup/CleanupPresetPanel.fxml

```diff
@@ -30,8 +30,10 @@
           <Insets left="20.0" />
        </VBox.margin>
       </VBox>
-      <CheckBox fx:id="cleanUpBiblatex" text="%Convert to biblatex format (e.g., store publication date in date field)" />
-      <CheckBox fx:id="cleanUpBibtex" text="%Convert to BibTeX format (e.g., store publication date in year and month fields)" />
+      <ToggleGroup fx:id="conversionToggleGroup">
```

Comment:
You need to define the toggle group first, like the timestampCleanup

Reply:
I'm sorry. I will revise it as soon as possible